.github/workflows/create-test-report.yml

```diff
@@ -28,8 +24,8 @@ jobs:
           reporttypes: Html
```

Comment:
We need to check whether the buildtoolkit puts the coverage reports in the same format in the dir provided in `reports: ...`
Don't know whether your want to do that before merging, otherwise would be great if you note that somewhere

Reply:
Ahh, I don't know why I wrote format... What i actually wanted to say is that we need to check that the reports are placed under the same path by the buildtoolkit that is given here, i.e. `artifacts/test-results/**/TestResults/**/coverage.cobertura.xml`

Reply:
They are: https://github.com/MORYX-Industry/tools/pull/2/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R89

Reply:
Great!